examples/ErrorHandling/HandleExpandedTextAdPolicyViolations.php

```diff
@@ -0,0 +1,244 @@
+use Google\Ads\GoogleAds\V2\Resources\Ad;
+use Google\Ads\GoogleAds\V2\Resources\AdGroupAd;
+use Google\Ads\GoogleAds\V2\Services\AdGroupAdOperation;
+use Google\Ads\GoogleAds\V2\Services\AdGroupAdServiceClient;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\StringValue;
+use GPBMetadata\Google\Ads\GoogleAds\V2\Errors\PolicyFindingError;
+
+/**
+ * This example demonstrates how to request an exemption for policy violations of an expanded text
+ * ad. If the request somehow fails with exceptions that are not policy finding errors, the example
+ * will stops instead of trying sending an exemption request.
+ */
+class HandleExpandedTextAdPolicyViolations
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    const AD_GROUP_ID = 'INSERT_AD_GROUP_ID_HERE';
+
+    public static function main()
+    {
+        // Either pass the required parameters for this example on the command line, or insert them
+        // into the constants above.
+        $options = (new ArgumentParser())->parseCommandArguments([
+            ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::AD_GROUP_ID => GetOpt::REQUIRED_ARGUMENT
+        ]);
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID,
+                $options[ArgumentNames::AD_GROUP_ID] ?: self::AD_GROUP_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @param int $adGroupId the ad group ID to add a keyword to
```

Comment:
No keyword is added to the ad group in this example, I think you meant to write 'an expanded text ad to' instead

Reply:
Done.